### ISSUE TITLE
Kotlin project compile failed with AndroidX

### ISSUE THREAD

**Issue by stefanJi** [bumptech/glide#4115]
**Glide Version**: 4.8.0

**Issue details / Repro steps / Use case background**: 

1. run ./gradlew installDevDebug
2. :app:app:kaptDevDebugKotlin will failed
```
e:app/build/generated/source/kapt/devDebug/com/bumptech/glide/GeneratedAppGlideModuleImpl.java:33: error: cannot find symbol
      @NonNull Registry registry) {
       ^
  symbol:   class NonNull
  location: class GeneratedAppGlideModuleImpl
e:app/build/generated/source/kapt/devDebug/com/bumptech/glide/GeneratedAppGlideModuleImpl.java:44: error: cannot find symbol
  @NonNull
   ^
  symbol:   class NonNull
  location: class GeneratedAppGlideModuleImpl
....

FAILURE: Build failed with an exception.
```

*Because Generated classes imported android.support.xxx.*

**Project info**:

gradle.properties:

```
# Project-wide Gradle settings.

# for databinding
android.databinding.enableV2=true

# Disables R8 for all modules.
android.enableR8 = false

# androidx
android.useAndroidX=true
android.enableJetifier=true
```

app/build.gradle:
```
ext {
   dep = [
      glideCompiler  : "com.github.bumptech.glide:compiler:4.8.0"
   ]
}

dependencies {
    kapt dep.glideCompiler
}
```

# the strange thing is

My colleagues can compile successfully, but I can't. I tried to clean build directory, clean gradle's caches, but not work.

./gradlew --version:
```
------------------------------------------------------------
Gradle 5.5.1
------------------------------------------------------------

Build time:   2019-07-10 20:38:12 UTC
Revision:     3245f748c7061472da4dc184991919810f7935a5

Kotlin:       1.3.31
Groovy:       2.5.4
Ant:          Apache Ant(TM) version 1.9.14 compiled on March 12 2019
JVM:          1.8.0_131 (Oracle Corporation 25.131-b11)
OS:           Mac OS X 10.15.3 x86_64
```

**Comment by stefanJi** [comment]
still issue

**Comment by stefanJi** [comment]
Fixed by upgrade to v4.10.0. https://github.com/bumptech/glide/releases/tag/v4.10.0
